docs/adguard-for-android/solving-problems/battery.md

```diff
@@ -9,35 +9,19 @@ This article covers AdGuard for Android, a multifunctional ad blocker that prote
 
 :::
 
-Sometimes you may notice that, according to Android built-in statistics, AdGuard consumes a lot of traffic and/or battery resource.
+On Android devices running OS 7 and earlier, built-in statistics often attributed high data and/or battery usage to AdGuard. This was because AdGuard accounted for all the traffic it filtered from different apps. As a result, AdGuard's portion of total data usage and battery consumption was overstated, while that of other apps was understated.
```

Comment:
```suggestion
On Android devices running OS 7 and earlier, built-in statistics often attributed high data and/or battery usage to AdGuard. This was because AdGuard counted all the traffic it filtered from various apps. As a result, AdGuard's share of total data and battery usage was overstated, while other apps were understated.
```